apps/web/src/pages/CommunityInterests/CommunityTalentPage/CommunityTalentTable.tsx

```diff
@@ -233,17 +373,72 @@ const CommunityTalentTable = ({ title }: CommunityTalentTableProps) => {
     ),
   ] as ColumnDef<CommunityTalentTableCommunityInterestFragmentType>[];
 
+  const hasSelectedRows = selectedRows.length > 0;
+
   return (
     <Table<CommunityTalentTableCommunityInterestFragmentType>
       data={communityInterestData}
       caption={title}
       columns={columns}
       hiddenColumnIds={["community", "workEmail", "preferredLang"]}
       isLoading={fetching}
+      search={{
```

Comment:
Is this only meant to search through the Name (first name, last name) values of each row?

Reply:
I talked to @Jerryescandon and he said to add the email and processes.

Reply:
This is getting spun off into another issue: https://github.com/GCTC-NTGC/gc-digital-talent/issues/13120